spec/support/shared_examples/structured_controller.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.shared_examples "structure persister" do |resource_symbol, presenter_factory|
   describe "#structure" do
-    let(:user) { FactoryGirl.create(:user) }
+    let(:user) { FactoryGirl.create(:admin) }
```

Comment:
This spec needs a closer look. Definitely some sipity-related permissions issues.